tests/specs/lemmas.k

```diff
@@ -57,6 +57,9 @@ module LEMMAS
     rule #padToWidth(32, #asByteStack(V)) => #buf(32, V)
       requires #rangeUInt(256, V)                                                                         [simplification]
 
+    rule #asWord(BUF1 ++ BUF2) >>Int LSHIFT => #asWord(BUF1)
+      requires #sizeByteArray(BUF2) *Int 8 ==Int LSHIFT                                                   [simplification]
```

Comment:
Wouldn't it be more appropriate to name this variable `RSHIFT` and not `LSHIFT`?

Reply:
Right, renamed.